docs/authentication/jwt-auth.html

```diff
@@ -16,13 +16,407 @@
+           <li>iat: The time the JWT was issued. Can be used to determine the age of the JWT</li>
+           <li>jti: Unique identifier for the JWT. Can be used to prevent the JWT from being replayed.</li>
+         </ul>
+    <p>SwiftJWT comes with a struct representing these Registered Claims which we will use for our example.</p>
+    <p>Beneath the users credentials, initialize the users claims:</p>
+    <pre><code class="language-swift">let myClaims = ClaimsStandardJWT(iss: "Kitura", sub: credentials.username, exp: Date(timeIntervalSinceNow: 3600))</code></pre>
+    <p>These claims tell us users username, that they were authenticated by Kitura and we have set the token to expire in one hour.</p>
+    <p>Beneath our claims we initialize our JWT:</p>
+    <pre><code class="language-swift">var myJWT = JWT(claims: myClaims)</code></pre>
+    <p>We can sign this JWT using the JWTSigner we created in step 2:</p>
+    <pre><code class="language-swift">let signedJWT = try myJWT.sign(using: App.jwtSigner)</code></pre>
+    <p>Finally we return the signed JWT String to the User:</p>
+    <pre><code class="language-swift">response.send(signedJWT)</code></pre>
+    <p>Our completed login route should look as follows:</p>
+    <pre><code class="language-swift">app.router.post("/jwtlogin") { request, response, next in
+    let credentials = try request.read(as: UserCredentials.self)
+    // Users credentials are authenticated
+    let myClaims = ClaimsStandardJWT(iss: "Kitura", sub: credentials.username, exp: Date(timeIntervalSinceNow: 3600))
+    var myJWT = JWT(claims: myClaims)
+    let signedJWT = try myJWT.sign(using: App.signer)
+    response.send(signedJWT)
+    next()
+}</code></pre>
+<div class="underline"></div>
+
+<h2 class="heading-2"><span class="blue-text">Step 6:</span> Test the JWT creation</h2>
+<p>Compile your project and start the server.</p>
+<p>To test the route using curl, open Terminal and enter the following:</p>
+<pre><code>curl -X POST \
+http://localhost:8080/jwtlogin \
+-H 'content-type: application/json' \
+-d '{
+"username": "Joe Bloggs",
+"password": "password"
+}'</code></pre>
+<p>You should be returned a JWT String that is structured `xxxx.yyyy.zzzz`
+     where xxxx is the base64 encoded header, yyyy is the base 64 encoded claims and zzzz is the signature.</p>
+<p>The following is a JWT generated using hs256 and the password "kitura":<p>
+<pre><code>eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJLaXR1cmEiLCJzdWIiOiJKb2UgQmxvZ2dzIiwiZXhwIjoxNTUzMDE4Mjg0LjMyOTcwMTl9.t55WealACtYGCQGS3EQgRQuurmNSBO5fWZqzqJjEIi</code></pre>
+<p>You can decode your JWT string to view the headers and claims at <a href="https://jwt.io/#debugger" rel="noopener" target="_blank">JWT.io</a></p>
+    <div class="underline"></div>
+
+    <h2 class="heading-2"><span class="blue-text">Step 7:</span> Verify a JWT</h2>
+    <p class="block-text">We have created a signed JWT for a user to authenticate themselves.
```

Comment:
How about "So far, we have created a signed JWT, which allows a user to authenticate themselves."